<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>청소당번정하기 - 메인</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>-15</number>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_totalnum">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>전체 인원</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_want">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>뽑을 인원</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕 ExtraBold</family>
      <pointsize>60</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>랜덤 번호 뽑기</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_description">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>무작위 번호를 정해보세요.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_madeby">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>520</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Made by. 10216 이지원</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_totalnum">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>전체 인원 수를 입력하세요.</string>
    </property>
    <property name="text">
     <string>21</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_want">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>정할 당번 수를 입력하세요.</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalcount">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕 Light</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>__명 중 __명을 무작위로 정합니다.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ok">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>버튼을 눌러 결과를 확인하세요.</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_version">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Version 2.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_split">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>961</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>____________________________________________________________________________________________________________</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_option">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>버튼을 눌러 결과를 확인하세요.</string>
    </property>
    <property name="text">
     <string>설정</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_skip">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>450</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>애니메이션 건너뛰기</string>
    </property>
   </widget>
   <zorder>label_want</zorder>
   <zorder>label_title</zorder>
   <zorder>label_description</zorder>
   <zorder>line_totalnum</zorder>
   <zorder>line_want</zorder>
   <zorder>label_totalcount</zorder>
   <zorder>label_madeby</zorder>
   <zorder>label_totalnum</zorder>
   <zorder>button_ok</zorder>
   <zorder>label_version</zorder>
   <zorder>label_split</zorder>
   <zorder>button_option</zorder>
   <zorder>checkBox_skip</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
